sass/application.scss

```diff
@@ -123,6 +123,13 @@ $govuk-font-family-print: 'GDS Transport';
   flex-basis: 80%;
 }
 
+.case-notes-omic-open {
+  display: none;
+}
+.case-notes-omic-open-warning {
+  margin-left: 15px;
```

Comment:
looks like the `govuk-grid-row` shifts the entire contents `15px` to the left, this reverses it so that in line with rest of content